README.md

```diff
@@ -19,13 +19,645 @@ A zero dependency library for serializing data.
 
 - [`JSON`](https://www.json.org/json-en.html)
 
-## [Usage](https://gamebridgeai.github.io/ts_serialize/)
+## Usage
 
-- [Installing](https://gamebridgeai.github.io/ts_serialize/installing)
-- [Serializable and SerializeProperty](https://gamebridgeai.github.io/ts_serialize/serializable)
-- [Strategies](https://gamebridgeai.github.io/ts_serialize/strategies)
-- [Global transformKey](https://gamebridgeai.github.io/ts_serialize/transforming)
-- [Polymorphism](https://gamebridgeai.github.io/ts_serialize/polymorphism)
```

Comment:
Fun fact, even if we don't include a table of contents ourselves one is still generated by github we can use:
![Screen Shot 2022-03-16 at 15 12 21](https://user-images.githubusercontent.com/49878815/158668891-7ecce049-f955-49be-988c-1d69c52d9552.png)

Reply:
🔥